benchmark/src/main/resources/benchmark.xsd

```diff
@@ -587,6 +587,9 @@
           <xs:element minOccurs="0" name="moveCountLimit" type="xs:long"/>
                               
           
+          <xs:element minOccurs="0" name="unimprovedBestSolutionLimit" type="xs:double"/>
```

Comment:
We need a better name for this one - one that actually indicated what is going to happen.

- `automaticExhaustionDetection` (we already use "exhaust" for exhaustive search, so maybe not)
- `automaticDepletionDetection`
- ... ?